src/Core/Math/MathExtented.js

```diff
@@ -78,4 +78,52 @@ MathExt.arrayRadToDeg = function(arrayDeg)
 	}
 };
 
+// TODO: Function in test :
+MathExt.step = function(val,stepVal)
+{
+    if(val<stepVal)
+        return 0.0;
+    else
+        return 1.0;
+
+};
```

Comment:
Coding style:
  * `{` on the same line
  * always use `{`, even for single line blocks
  * remove extra blank lines